environment.yml

```diff
@@ -11,6 +14,4 @@ dependencies:
   - cmake
   - gmp
   - numpy
-  - python
-  - pip:
-    - pythran
+  - python==3.7
```

Comment:
Why do you need to pin the version?

Reply:
just I try for eliminating the error, I'll revert this. thanks